### PR TITLE
Removing word "Incubating" from project

### PR DESCRIPTION
**Pull request by lukasnalezenec**

### CODE REVIEW

File: parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoReadSupport.java [hunk 1]
```diff
@@ -46,6 +46,16 @@ public static void setRequestedProjection(Configuration configuration, String re
...
 
+  /**
+   * Set name of protobuf class to be used for reading data.
+   * If no class is set, value from file header is used.
+   * Note that the value in header is present only if the file was written
+   * using parquet-protobuf project, it will fail otherwise.
+   * */
+  public static void setProtobufClass(Configuration configuration, String protobufClass) {
+    configuration.set(PB_CLASS, protobufClass);
+  }
```
Comment (julienledem) [inline parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoReadSupport.java]: is this an unrelated change?

File: parquet-protobuf/src/test/java/org/apache/parquet/proto/ProtoInputOutputFormatTest.java [hunk 2]
```diff
@@ -96,6 +93,34 @@ public void testProjection() throws Exception {
   }
 
   /**
+   * When user specified protobuffer class in configuration,
+   * It should replace class specified in header.
+   * */
+  @Test
+  public void testCustomProtoClass() throws Exception {
```
Comment (julienledem) [inline parquet-protobuf/src/test/java/org/apache/parquet/proto/ProtoInputOutputFormatTest.java]: same question

**Comment by julienledem** [comment]
overall this looks good to me except for the unrelated change.
please get PR reviewed by another committer before merging.
otherwise, thanks for cleaning up.

**Comment by lukasnalezenec** [comment]
Hi, 
I have closed this PR without merging because Ryan Blue opened similar pull request 186 few hours after this one. The unrelated changes were caused by wrong git branching.

 https://github.com/apache/parquet-mr/pull/186
